### ISSUE TITLE
Provided solution can't find some projects

### ISSUE THREAD

**Issue by raysan5** [ChrisDill/Raylib-cs#37]
I tried downloading the repo and open provided solution, most of the projects contained can not be found. I tried adding `Tests.MultiTarget.csproj` but it fails on compilation, I get this error:
```
Error	NU1101	Unable to find package Microsoft.DotNet.ILCompiler. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages, nuget.org
```

**Comment by ChrisDill** [comment]
I think the DotNet.ILCompiler was needed for CoreRT although I have not spent much time with it. Removed for now. Will try to revisit CoreRT support fully in the future.

A while back I changed Tests to a multi-target project so I didn't need to have seperate project files for each target. Updated just now with some of the changes.

I have not tested in visual studio etc so I don't know how to specify which framework it should build in. However I can specify and build using this command.

```
dotnet run --framework=netcoreapp3.1 --project=Tests/Tests.MultiTarget.csproj
```

Let me know if this helps.
